iron-form.html

```diff
@@ -343,7 +342,9 @@
     },
 
     _registerElement: function(e) {
-      var element = e.target;
+      // Get the actual element that fired the event
+      var element = Polymer.dom(e).path[0];
+
       element._parentForm = this;
       this._customElements.push(element);
```

Comment:
:)

We need it for https://github.com/PolymerElements/iron-form-element-behavior/blob/master/iron-form-element-behavior.html#L79, so that when an element is detached, you can still unregister from the form. The problem is that `detached` is called *after* you've been removed from the form, so you have no way to get back to it and tell it to unregister you